src/Driver/PgSQL/Connection.php

```diff
@@ -47,6 +48,15 @@ public function __construct($connection)
         $this->parser     = new Parser(false);
     }
 
+    public function __destruct()
+    {
+        if (isset($this->connection)) {
+            @pg_close($this->connection);
+        }
+
+        unset($this->connection);
```

Comment:
Is there a point to this call to `unset`? The connection object is being destroyed, surely all of its properties will be destroyed as well?

Reply:
You're right, it's probably not necessary. I'll check.

Reply:
Removed.